packages/hoppscotch-common/src/components/app/Header.vue

```diff
@@ -281,8 +282,22 @@ const showTeamsModal = ref(false)
 const breakpoints = useBreakpoints(breakpointsTailwind)
 const mdAndLarger = breakpoints.greater("md")
 
+const bannerService = useService(BannerService)
+const banner = bannerService.content
 const network = reactive(useNetwork())
 
+watch(
+  network,
+  () =>
+    (banner.value = network.isOnline
+      ? undefined
+      : {
+          type: "info",
+          text: () => t("helpers.offline"),
+          alternateText: () => t("helpers.offline_short"),
```

Comment:
same as the last comment.

```suggestion
          alternateText: t("helpers.offline_short"),
```